optuna/importance/_fanova/_fanova.py

```diff
@@ -73,58 +72,40 @@ def fit(
 
         self._trees = [_FanovaTree(e.tree_, search_spaces) for e in self._forest.estimators_]
         self._column_to_encoded_columns = column_to_encoded_columns
-        self._variances = {}
 
         if all(tree.variance == 0 for tree in self._trees):
             # If all trees have 0 variance, we cannot assess any importances.
             # This could occur if for instance `X.shape[0] == 1`.
             raise RuntimeError("Encountered zero total variance in all trees.")
 
-    def get_importance(self, features: Tuple[int, ...]) -> Tuple[float, float]:
+    def get_importance(self, feature: int) -> Tuple[float, float]:
```

Comment:
Seems fair 👍  Just for the record, we used to accept tuples in order to allow/check the computations for pairwise importances. But it's not available through public APIs for the time being and we can reintroduce the logic whenever needed.